packages/core/tests/unit/models/bard-request/fragments/filter-test.js

```diff
@@ -30,7 +30,7 @@ module('Unit | Model Fragment | BardRequest - Filter', function(hooks) {
   setupTest(hooks);
 
   hooks.beforeEach(async function() {
-    Server = setupMock();
+    Server = startMirage();
```

Comment:
Can we use `setupMirage(hooks)` here?

Reply:
And use `this.server` instead of `let Server`

Reply:
Yea we could switch everything to using `setupMirage(hooks)`, I just wasn't a fan of the `this.server` being magically set from [here](https://github.com/samselikoff/ember-cli-mirage/blob/c77f1997701fbc476cade1717197d48a0bcd70b9/addon-test-support/setup-mirage.js#L13)

Reply:
> I just wasn't a fan of the this.server being magically set from here

Yeah, I'm with you, but I think automatic `this.server.shutdown()` via hooks is worth the trade-off.